src/Kernel/ConstantBlockClosure.class.st

```diff
@@ -38,11 +38,24 @@ ConstantBlockClosure class >> numArgs: numArgs literal: aLiteral [
 		literal: aLiteral
 ]
 
+{ #category : #literals }
+ConstantBlockClosure >> blockViaLiteralPath: anArray [ 
+	"return the compiledBlock for a path (see CompiledBlock>>literalPath)"
+	^ self
```

Comment:
I'm afraid that always retuning self with cause hard to track bugs if a wrong path is given.
I like exact result and early failures.

`anArray isEmpty ifTrue: [ ^self ] ifFalse: [ raise some index error or something ]`

Reply:
ok